libsolidity/codegen/MultiUseYulFunctionCollector.h

```diff
@@ -25,10 +25,21 @@
 #include <functional>
 #include <map>
 #include <string>
+#include <vector>
 
 namespace solidity::frontend
 {
 
+struct YulUtilFunction
+{
+	std::string name;
```

Comment:
Having the name here is redundant now that `createFunction()` as the `_name` parameter again and stores it in `m_requestedFunctions`.